src/networks.js

```diff
@@ -13,6 +13,17 @@ module.exports = {
     scriptHash: 0x05,
     wif: 0x80
   },
+  regtest: {
+    messagePrefix: '\x18Bitcoin Signed Message:\n',
+    bech32: 'bcrt',
```

Comment:
Any reference for this by the way?

Reply:
It is not mentioned in BIP173  because it is Bitcoin Core specific. The only reference I can find is the code itself https://github.com/bitcoin/bitcoin/blob/master/src/chainparams.cpp#L341

Reply:
It's also mentioned here https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Thanks @Janaka-Steph